websites/A/AnimeTvOnline/presence.ts

```diff
@@ -0,0 +1,44 @@
+const presence = new Presence({
+	clientId: "1042875166021124229",
+});
+
+enum Assets {
+	logo = "https://i.imgur.com/b0PVWvC.png"
+}
```

Comment:
Asset names should be capitalized. Also, with so few assets, an enum may not be needed.
```suggestion
enum Assets {
	Logo = "https://i.imgur.com/b0PVWvC.png"
}
```

Reply:
Planning on updating this with new features, an enum would be the best move right now.